torch/_inductor/config.py

```diff
@@ -80,11 +79,12 @@
 
 
 def is_fbcode():
-    import torch
-
     return not hasattr(torch.version, "git_version")
 
 
+# warnings intended for PyTorch developers, disable for point releases
+developer_warnings = is_fbcode() or "+" in torch.__version__
```

Comment:
i think this is ok. does __version__ always exist or are there some cases where it isn't available?

Reply:
Seems always set to me:
https://github.com/pytorch/pytorch/blob/7c44823a4e0506b2eb7f1aa667ba939ba9d56969/torch/__init__.py#L26-L29